alpaca/data/historical/stock.py

```diff
@@ -94,15 +84,9 @@ def get_stock_bars(
         Returns:
             Union[BarSet, RawData]: The bar data either in raw or wrapped form
         """
-
-        params = request_params.to_request_fields()
-
-        # paginated get request for market data api
-        raw_bars = self._data_get(
-            endpoint_data_type="bars",
-            endpoint_asset_class="stocks",
-            api_version="v2",
-            **params,
+        raw_bars = self.get_marketdata(
+            path="/stocks/bars",
```

Comment:
I'm happy you pointed this out, I thought about it as well. With this change now we'll return `{}` instead of `{"AAPL": []}` if there are no data points in the single symbol endpoint. In my opinion this is acceptable, especially that if you provide multiple symbols, this is already the behaviour. If you disagree, let's talk about this.

Reply:
@gnvk Thanks! I also thought the breaking change should be ok.
I just wanted to confirm to write this change on the release note.